### PR TITLE
fixate 1.14.5

### PR DESCRIPTION
**Pull request by patricklodder**
Marks 1.14.5 as release (and fixes the major version number in `clientversion.cpp`.)

Can be merged AFTER all outstanding items from #2655 are done.

### CODE REVIEW

**Review by rnicoll — APPROVED** [review]
Yay!

File: configure.ac [hunk 1]
```diff
@@ -4,7 +4,7 @@ define(_CLIENT_VERSION_MAJOR, 1)
 define(_CLIENT_VERSION_MINOR, 14)
 define(_CLIENT_VERSION_REVISION, 5)
 define(_CLIENT_VERSION_BUILD, 0)
-define(_CLIENT_VERSION_IS_RELEASE, false)
+define(_CLIENT_VERSION_IS_RELEASE, true)
```
Comment (Bartascnk) [inline configure.ac:7]: Prawda

Comment (Zhila136) [inline configure.ac:7]: _CLIENT_VERSION_IS_RELEASE

**Review by Zhila136 — COMMENTED** [review]
0xba2ae424d960c26247dd6c32edc70b295c744c43

**Review by Ikaberry202 — COMMENTED** [review]
Good to see this

**Pull request merged**